content/en/docs/contribute/style/style-guide.md

```diff
@@ -631,6 +631,10 @@ You can remove ... | You can easily remove ...
 These steps ... | These simple steps ...
 {{< /table >}}
 
+### EditorConfig
+EditorConfig is useful for maintaining a consistent coding style across all the editors , this helps the developers using different editors (like vim , emacs , atom , vscode etc) would be able to contribute to the project with the same style.usually for doing this an .editorconfig file is used. 
+
+to check out the .editorconfig click [here](https://github.com/kubernetes/website/blob/main/.editorconfig) 
```

Comment:
These sentences don't flow well. Please check.

Reply:
Maybe a reword to:

```suggestion
The Kubernetes project maintains an EditorConfig file that sets common style preferences in text editors
such as VS Code.  You can use this file if you want to improve how consistent your contributions are with
the rest of the project. To download the file, refer to
[`.editorconfig`](https://github.com/kubernetes/website/blob/main/.editorconfig).
```

Reply:
I like this phrasing better @shannonxtreme, but it may be odd to ask folks to download the file since it's a part of the k/website repo.

It may be better to refer to the file's location. Or mention the .editorconfig file in the body of the paragraph with the link to it?

Reply:
Good point. To view the file, refer to .editorconfig in the repository root?

Reply:
Hi @Karthik-PM . What do you think about incorporating the suggested edits?